src/ui/public/utils/zoom_to_precision.js

```diff
@@ -0,0 +1,26 @@
+import { geohashColumns } from 'ui/utils/decode_geo_hash';
+
+const maxPrecision = 12;
+/**
+ * Map Leaflet zoom levels to geohash precision levels.
+ * The size of a geohash column-width on the map should be at least `minGeohashPixels` pixels wide.
+ */
+const zoomPrecision = {};
+const minGeohashPixels = 16;
+for (let zoom = 0; zoom <= 21; zoom += 1) {
```

Comment:
good point. if we add zoom levels beyond 21, this is going to cause issues indeed